components/brave_welcome_ui/components/screens/searchBox.tsx

```diff
@@ -18,11 +19,62 @@ interface Props {
   index: number
   currentScreen: number
   onClick: () => void
+  isDefaultSearchGoogle: boolean
 }
 
-export default class SearchEngineBox extends React.PureComponent<Props, {}> {
+interface State {
+  searchEngineSelected: boolean,
+  searchEngineIndex: number,
+  searchProviders: Array<any>
```

Comment:
@imptrx do you think we can improve this type? seems an array of objects with known properties